draft-barnes-acme.md

```diff
@@ -1093,6 +1093,17 @@ type (required, string):
 To validate a DNS challenge, the server queries for TXT records under the validation domain name.  If it receives a record whose contents match the token in the challenge, then the validation succeeds.  Otherwise, the validation fails.
 
 
+## Wildcard Domain Validation
+
+When the identifier being validated is a domain name, it may contain a wildcard character "\*" as its left-most label (e.g., "\*.foo.example.com"). A wildcard certificate contains a wildcard domain in a subjectAltName of type DNS-ID and automatically vouches for any and all host names within the bare domain name (the FQDN to the immediate right of the wildcard label).
```

Comment:
@jsha If I'm not mistaken, the current implementation is as follows:
1. Client enters the identifier they want to get validated
2. Server issues challenges
3. Client does challenges, server checks answers and issues certificate for the identifier.

The steps below would happen between (1) and (2). Are you suggesting that wildcard domain handling get pushed later?